### DISCUSSION TITLE
I think this is the One!

### DISCUSSION THREAD

**Discussion by jbeckton** [hasura/graphql-engine#8984] [General] [answered]
I have been looking at a lot of tech over the last year. I am trying to build a custom platform for an online business idea. Several months ago I decided I needed to find ways to build faster. I started looking at some ways to shortcut and reduce my efforts by not coding my API, or atleast most of it. I only had a couple parameters to go by in my search and one of the most important was sticking with tried and true DB platforms like MongoDB or Postgres and reduce vendor lock in risks as much as possible. I have been reading Hasura docs for 2 days and I think this is the tech I have been looking for.

Max Kudos to the Hasura team!

**Comment by adas98012** [comment, marked as answer]
Thank you for sharing your positive experience with Hasura GraphQL Engine. We do have an active community of Hasura Users on [Discord Server](https://discord.com/invite/hasura). Please join the community and share your knowledge and experience.
